console/src/main/webapp/WEB-INF/web.xml

```diff
@@ -145,6 +146,15 @@
       <servlet-name>GwtSettingsServiceImpl</servlet-name>
       <url-pattern>/settings/*</url-pattern>
    </servlet-mapping>
+   
+   <servlet>
```

Comment:
Why this new servlet definition? `GwtAccessInfoServiceImpl` is already registered at 222